cmake/scripts/linux/Install.cmake

```diff
@@ -314,3 +314,11 @@ if(CPACK_GENERATOR)
     message(FATAL_ERROR "DEB Generator: Can't configure CPack to generate Debian packages on non-linux systems.")
   endif()
 endif()
+
+if(CORE_PLATFORM_NAME_LC STREQUAL gbm)
+  if(SYSTEMD_FOUND)
```

Comment:
where is this set? Am I missing the optional dependency definition and the Find module?